db/views/active_providers_v01.sql

```diff
@@ -0,0 +1,24 @@
+with submissions as (
+  select
+  	application.application_type,
+    application_version.application -> 'firm_office' ->> 'account_number' as office_code,
+    application_version.application -> 'firm_office' ->> 'name' as firm_name,
+    date_trunc('week', application_version.created_at) as submitted_start
+  from application_version join application on application_version.application_id = application.id
+  where version = 1 -- submission version only
+  group by 1, 2, 3, 4
```

Comment:
I was today years old when I learnt that you can reference columns as numbered indexes like this!

Reply:
that said, what is the reason for using this syntax when we explicitly reference the column names later on - feels harder to read

Reply:
Agree I prefer the otehr syntax as easier to read... but in this situation when they are so complex this limits a heap of noise...

i.e.
```sql
  select
  	application.application_type,
    application_version.application -> 'firm_office' ->> 'account_number' as office_code,
    application_version.application -> 'firm_office' ->> 'name' as firm_name,
    date_trunc('week', application_version.created_at) as submitted_start
  from application_version join application on application_version.application_id = application.id
  where version = 1 -- submission version only
  group by
  	application.application_type,
    application_version.application -> 'firm_office' ->> 'account_number' as office_code,
    application_version.application -> 'firm_office' ->> 'name' as firm_name,
    date_trunc('week', application_version.created_at) as submitted_start
```
so inclinded to take the hit on it being slightly different in this case, but for a good reason